website/src/pages/v6/all-props.mdx

```diff
@@ -165,7 +165,7 @@ accessibility, clipping, or z-index issues.
 ```js
 tippy(targets, {
   // default (takes reference as an argument)
```

Comment:
I see it is mentioned here but I think this can be made more explicit in the signature